looker-sdk.gemspec

```diff
@@ -24,5 +24,5 @@ Gem::Specification.new do |s|
   s.require_paths = %w(lib)
   s.add_dependency 'jruby-openssl' if s.platform == :jruby
   s.add_dependency 'sawyer', '~> 0.8'
-  s.add_dependency 'faraday', ['>= 1.0', '< 2.0']
+  s.add_dependency 'faraday', ['>= 1.7.1', '< 2.0']
```

Comment:
I just realized this bump in minimal version - does pre-1.7.1 not work with the `http.headers[:authorization]` change?

Reply:
I confirmed it works with 1.2.0 and later, so I set 1.2.0 as the min.